test/gtest/uct/ib/test_rc.cc

```diff
@@ -666,7 +666,13 @@ void test_rc_flow_control::test_general(int wnd, int soft_thresh,
     flush();
 }
 
-void test_rc_flow_control::test_pending_grant(int wnd, uint64_t *wait_fc_seq)
+void test_rc_flow_control::wait_fc_hard_resend(entity *e)
+{
+    UCS_TEST_ABORT("waiting FC_HARD_REQ resend isn't implemented in RC");
+}
+
+void test_rc_flow_control::test_pending_grant(int16_t wnd,
+                                              bool wait_fc_hard_req_resend)
```

Comment:
> BTW is it still needed? given that we ++ tx.seq unconditionally

yes, it fixes the test, because we don't increment FC entry anymore if it already exists in FC hash

Reply:
> why need bool? virt function could be empty..

not sure that I understand, functions should be identical and should have the same set of parameters, shouldn't it?

Reply:
1. why can't we wait for iface->tx.fc_seq to change? why must wait on hash entry?
2. instead of aborting the test with RC, can we just make the RC virtual function empty?

Reply:
> why can't we wait for iface->tx.fc_seq to change? why must wait on hash entry?

because `iface->tx.fc_seq` won't be changed in case of sending `FC_HARD_REQ` second time

> instead of aborting the test with RC, can we just make the RC virtual function empty?

it doesn't make sense to run this test for RC, so I decided to abort the test if it will be tested by mistake.